Packs/Akamai_WAF/Integrations/Akamai_WAF/Akamai_WAF.py

```diff
@@ -5843,6 +6256,617 @@ def list_cps_active_certificates_command(
+            delay). urllib3 will sleep for::
+
+                {backoff factor} * (2 ** ({number of total retries} - 1))
+
+            seconds. If the backoff_factor is 0.1, then :func:`.sleep` will sleep
+            for [0.0s, 0.2s, 0.4s, ...] between retries. It will never be longer
+            than :attr:`Retry.BACKOFF_MAX`.
+
+            By default, backoff_factor set to 5
+
+        :type raise_on_redirect ``bool``
+        :param raise_on_redirect: Whether, if the number of redirects is
+            exhausted, to raise a MaxRetryError, or to return a response with a
+            response code in the 3xx range.
+
+        :type raise_on_status ``bool``
+        :param raise_on_status: Similar meaning to ``raise_on_redirect``:
+            whether we should raise an exception, or return a response,
+            if status falls in ``status_forcelist`` range and retries have
+            been exhausted.
+
+        :type empty_valid_codes: sinlge integer or ``comma separated integers``
+        :param empty_valid_codes: A list of all valid status codes of empty responses (usually only 204, but
+            can vary)
+
+        :type with_metrics ``bool``
+        :param with_metrics: Whether or not to calculate execution metrics from the response
+
+    Returns:
+        human readable (markdown format), entry context and raw response
+    """
+
+    if headers is not None and not isinstance(headers, dict):
+        headers = json.loads(headers)
+    if params is not None and not isinstance(params, dict):
+        params = json.loads(params)
+    if data is not None and not isinstance(data, dict):
+        data = json.loads(data)
+    if json_data is not None and not isinstance(json_data, dict):
+        json_data = json.loads(json_data)
+    if files is not None and not isinstance(files, dict):
+        files = json.loads(files)
+    if timeout is not None:
+        if "," in timeout:
+            timeout = tuple([float(x) for x in timeout.split(",")])
+        else:
+            timeout = float(timeout)
+    if ok_codes is not None:
+        if "," in ok_codes:
+            ok_codes = tuple([int(x) for x in ok_codes.split(",")])
+    if retries is not None:
+        retries = int(retries)
+    if status_list_to_retry is not None:
+        if "," in status_list_to_retry:
+            status_list_to_retry = [int(x) for x in status_list_to_retry.split(",")]
+        else:
+            status_list_to_retry = [int(status_list_to_retry)]
+    if backoff_factor is not None:
+        backoff_factor = float(backoff_factor)
+    if raise_on_redirect is not None:
+        raise_on_redirect = argToBoolean(raise_on_redirect)
+    if raise_on_status is not None:
+        raise_on_status = argToBoolean(raise_on_status)
+    if empty_valid_codes is not None:
+        if "," in empty_valid_codes:
+            empty_valid_codes = [int(x) for x in empty_valid_codes.split(",")]
+        else:
+            empty_valid_codes = [int(empty_valid_codes)]
```

Comment:
```suggestion
        empty_valid_codes = [int(x) for x in empty_valid_codes.split(",")]
```
Are we simply expecting a single integer/comma seperated integers? This should work regardless in that case

Reply:
Yes, that works. Thanks!